react/ProductHighlights.test.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { render } from '@vtex/test-tools/react'
-import { useProduct, ProductContext } from 'vtex.product-context'
+import type { ProductContext } from 'vtex.product-context'
+import { useProduct } from 'vtex.product-context'
```

Comment:
To make the bot stop complaining

```suggestion
import { useProduct, ProductContext } from 'vtex.product-context'
```